### ISSUE TITLE
WARNING: Invalid body length for classification lookup, not parsing.

### ISSUE THREAD

**Issue by jkspring** [noaa-rsd/cBLUE.github.io#41]
Get the following warning when running certain las files through cBLUE.

_WARNING: Invalid body length for classification lookup, not parsing.
WARNING: Invalid body length for classification lookup, not parsing._

Does not seem to be an issue with the output, just a warning.

**Comment by forkozi** [comment]
I agree...from what I can tell, this doesn't affect LiDOG.  It has to do with the self.rec_len_after_header attribute of VLR record_id 0 (https://github.com/laspy/laspy/blob/769bfe2cd4d0fc265c92a6182cb12107e28c7074/laspy/header.py#L81).  I'm not sure where this originates.  Should we inquire with the contractor, or leave it be since it doesn't seem to affect anything?
